synapse/handlers/appservice.py

```diff
@@ -261,14 +267,15 @@ async def _handle_presence(
                     ),
                 }
                 for event in presence_events
-            ]
-            events = events + presence_events
+            )
+
+        return events
```

Comment:
@Half-Shot I believe this is an bug in the original code?

Reply:
Yeah, ugh.

Reply:
This got uplifted to 1.22.0rc2 in #8656.